frontend/amundsen_application/api/healthcheck.py

```diff
@@ -0,0 +1,47 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+from http import HTTPStatus
+import json
+from requests.exceptions import ConnectionError, ReadTimeout
+from typing import Tuple
+
+from amundsen_common.models.api import health_check
+from flask import current_app as app
+
+from amundsen_application.api.utils.request_utils import request_metadata, request_search
+
+
+def run_healthcheck() -> Tuple[str, int]:
+    search_url = app.config['SEARCHSERVICE_BASE'] + '/healthcheck'
+    metadata_url = app.config['METADATASERVICE_BASE'] + '/healthcheck'
+
+    try:
+        search_health_resp = request_search(url=search_url, method='GET')
+        if search_health_resp.status_code == HTTPStatus.OK:
+            search_health = health_check.HealthCheck(**search_health_resp.json())
+        else:
+            search_checks = {'search_service': {'status': 'Could not get health from search'}}
+            search_health = health_check.HealthCheck(status=health_check.FAIL, checks=search_checks)
+    except (ConnectionError, ReadTimeout):
+        search_checks = {'search_service': {'status': 'Unable to connect.'}}
+        search_health = health_check.HealthCheck(status=health_check.FAIL, checks=search_checks)
+
+    try:
```

Comment:
this block is 99% the same as the one for search. do you think that in the spirit of DRY it could be rewritten as a function?

Reply:
yep, that will work better if/when we add additional checks